### PR TITLE
chore(button-legacy): diverse components and pages

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-1071

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [ ] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by RemiBonnet** [comment]
[Qovery Preview](https://www.qovery.com)
---

Qovery can create a Preview Environment for this PR.
To trigger its creation, please post a comment with one of the following command.

| Command | Blueprint environment |
|---|---|
| /qovery []() preview cc1de7de-94e5-40f8-8e45-bc8986d9dfec | [storybook](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/cc1de7de-94e5-40f8-8e45-bc8986d9dfec/services/general) |
 /qovery []() preview 28c47145-c8e7-4b9d-8d9e-c65c95b48425 | [staging](https://console.qovery.com/organization/3d542888-3d2c-474a-b1ad-712556db66da/project/a64f6cc1-4e34-433b-b8fd-5359716fb01b/environment/28c47145-c8e7-4b9d-8d9e-c65c95b48425/services/general) |
| /qovery []() preview {all\|UUID1,UUID2,...} | To preview multiple environments |

---

This comment has been generated from [Qovery](https://qovery.com) AI :robot:.
Below, a word from its wisdom :
> You are going to thank you later if it contains tests

File: libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx [hunk 1]
```diff
@@ -40,39 +42,44 @@ export function ButtonsActionsLogs(props: ButtonsActionsLogsProps) {
...
-        className="btn btn-icon btn-icon--small btn-icon--dark"
-        onClick={(event) => downloadJSON(event)}
+        type="button"
+        color="neutral"
+        className="w-10 justify-center"
+        onClick={() => downloadJSON()}
       >
         <Icon iconName="cloud-arrow-down" />
-      </a>
+      </Button>{' '}
```
Comment (ctjhoa) [inline libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx:82]:
```suggestion
      </Button>
```

File: libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx [hunk 2]
```diff
@@ -40,39 +42,44 @@ export function ButtonsActionsLogs(props: ButtonsActionsLogsProps) {
...
-              size={ButtonLegacySize.TINY}
-              style={!pauseLogs ? ButtonIconStyle.DARK : ButtonIconStyle.BASIC}
+            <Button
+              className="w-10 justify-center"
+              type="button"
+              color={!pauseLogs ? 'neutral' : 'brand'}
               onClick={() => setPauseLogs(!pauseLogs)}
-            />
+            >
+              <Icon iconName={!pauseLogs ? 'pause' : 'play'} />
```
Comment (ctjhoa) [inline libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx:51]:
```suggestion
              <Icon iconName={pauseLogs ? 'play' : 'pause'} />
```

File: libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx [hunk 3]
```diff
@@ -40,39 +42,44 @@ export function ButtonsActionsLogs(props: ButtonsActionsLogsProps) {
...
           <div>
-            <ButtonIcon
-              dataTestId="pause-button"
-              icon={!pauseLogs ? IconAwesomeEnum.PAUSE : IconAwesomeEnum.PLAY}
-              size={ButtonLegacySize.TINY}
-              style={!pauseLogs ? ButtonIconStyle.DARK : ButtonIconStyle.BASIC}
+            <Button
+              className="w-10 justify-center"
+              type="button"
+              color={!pauseLogs ? 'neutral' : 'brand'}
```
Comment (ctjhoa) [inline libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx:48]:
```suggestion
              color={pauseLogs ? 'brand' : 'neutral'}
```

File: libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx [hunk 4]
```diff
@@ -12,11 +12,13 @@ export interface ButtonsActionsLogsProps {
 export function ButtonsActionsLogs(props: ButtonsActionsLogsProps) {
   const { refScrollSection, data, pauseLogs, setPauseLogs } = props
 
-  const downloadJSON = (event: MouseEvent) => {
+  const downloadJSON = () => {
```
Comment (ctjhoa) [inline libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx:15]: you can use `downloadjs` directly

**Review by ctjhoa — CHANGES_REQUESTED** [review]

File: libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx (hunk 4 above)
Comment (RemiBonnet) [reply libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx:16]: https://github.com/Qovery/console/pull/1316/commits/5f553326bc98c897a74a5871c5eefb98ecbea565

File: libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx (hunk 3 above)
Comment (RemiBonnet) [reply libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx]: https://github.com/Qovery/console/pull/1316/commits/5f553326bc98c897a74a5871c5eefb98ecbea565

File: libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx (hunk 2 above)
Comment (RemiBonnet) [reply libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx]: https://github.com/Qovery/console/pull/1316/commits/5f553326bc98c897a74a5871c5eefb98ecbea565

File: libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx (hunk 1 above)
Comment (RemiBonnet) [reply libs/shared/console-shared/src/lib/layout-logs/buttons-actions-logs/buttons-actions-logs.tsx]: https://github.com/Qovery/console/pull/1316/commits/5f553326bc98c897a74a5871c5eefb98ecbea565

**Review by ctjhoa — APPROVED** [review]

**Pull request merged**